<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleEntryDialog</class>
 <widget class="QDialog" name="ScheduleEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule Entry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelStartTime">
       <property name="text">
        <string>Start time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="timeEditStart">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string notr="true">HH:mm</string>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEndTime">
       <property name="text">
        <string>End time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="timeEditEnd">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string notr="true">HH:mm</string>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDays">
       <property name="text">
        <string>Days:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxDays">
       <item>
        <property name="text">
         <string>Every day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekdays</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekends</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tuesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wednesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thursday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Friday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Saturday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sunday</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelProfile">
       <property name="text">
        <string>Speed profile:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxProfile"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScheduleEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScheduleEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
